motorparts/__init__.py

```diff
@@ -27,18 +27,20 @@
 TOKEN_URL = 'https://www.mopar.com/moparsvc/token'
 TOW_URL = 'https://www.mopar.com/moparsvc/vehicle/tow-guide/vin'
 VHR_URL = 'https://www.mopar.com/moparsvc/getVHR'
-REMOTE_COMMAND_URL = 'https://www.mopar.com/moparsvc/remoteCommands'
-REMOTE_STATUS_URL = 'https://www.mopar.com/moparsvc/vehicle/remote/status'
+REMOTE_LOCK_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/lock'
+REMOTE_ENGINE_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/engine'
+REMOTE_ALARM_COMMAND_URL = 'https://www.mopar.com/moparsvc/connect/alarm'
+global REMOTE_COMMAND_URL
```

Comment:
Any design should not have to use `global`. Modify `remote_command(...)` to check the `command` arg to select the correct const url, that's probably the easiest thing to do.

Reply:
I think I have it fixed now. apologize for any errors or style concerns. this is my first time using Python.